docs/deploying.md

```diff
@@ -101,53 +101,49 @@ and yaml files. `samples` folder includes yaml representation of sample CRDs.
 ```diff
 tree ironic-deployment/
 ironic-deployment/
-├── basic-auth
-│   ├── default
-│   │   ├── auth.yaml
-│   │   └── kustomization.yaml
-│   ├── ironic-auth-config-tpl
-│   ├── ironic-inspector-auth-config-tpl
-│   ├── keepalived
-│   │   ├── auth.yaml
-│   │   └── kustomization.yaml
-│   └── tls
-│       ├── default
-│       │   ├── auth.yaml
-│       │   └── kustomization.yaml
-│       └── keepalived
-│           ├── auth.yaml
-│           └── kustomization.yaml
-├── certmanager
-│   ├── certificate.yaml
-│   └── kustomization.yaml
+├── base
+│   ├── ironic.yaml
+│   └── kustomization.yaml
+├── components
+│   ├── basic-auth
+│   │   ├── auth.yaml
+│   │   ├── ironic-auth-config
+│   │   ├── ironic-auth-config-tpl
+│   │   ├── ironic-htpasswd
+│   │   ├── ironic-inspector-auth-config
+│   │   ├── ironic-inspector-auth-config-tpl
+│   │   ├── ironic-inspector-htpasswd
+│   │   └── kustomization.yaml
+│   ├── keepalived
+│   │   ├── ironic_bmo_configmap.env
+│   │   ├── keepalived_patch.yaml
+│   │   └── kustomization.yaml
+│   └── tls
+│       ├── certificate.yaml
+│       ├── kustomization.yaml
+│       ├── kustomizeconfig.yaml
+│       └── tls.yaml
 ├── default
-│   ├── ironic_bmo_configmap.env
-│   └── kustomization.yaml
-├── ironic
-│   ├── ironic.yaml
-│   └── kustomization.yaml
-├── keepalived
-│   ├── ironic_bmo_configmap.env
-│   ├── keepalived_patch.yaml
-│   └── kustomization.yaml
+│   ├── ironic_bmo_configmap.env
+│   └── kustomization.yaml
+├── overlays
+│   ├── basic-auth_tls
+│   │   ├── basic-auth_tls.yaml
+│   │   └── kustomization.yaml
+│   ├── basic-auth_tls_keepalived
+│   │   └── kustomization.yaml
+│   └── temp
```

Comment:
temp?

Reply:
Good find. I will add an explanation for this in the docs and probably remove it from here since it is a temporary folder (removed on `make clean`).